crates/pallet-balanced-currency-swap-bridges-initializer/src/upgrade_init.rs

```diff
@@ -9,76 +13,54 @@ use crate::{
     CURRENT_BRIDGES_INITIALIZER_VERSION,
 };
 
-/// Initialize the bridges pot accounts if required.
-pub fn on_runtime_upgrade<T: Config>() -> Weight {
-    let last_initializer_version = <LastInitializerVersion<T>>::get();
-    let last_force_rebalance_ask_counter = <LastForceRebalanceAskCounter<T>>::get();
-    let current_force_rebalance_ask_counter = T::ForceRebalanceAskCounter::get();
-
-    let mut weight = T::DbWeight::get().reads(2);
+/// Execute upgrade init.
+pub struct UpgradeInit<T>(sp_std::marker::PhantomData<T>);
```

Comment:
Why change this from migration to an explicit upgrade? Doesn't make sense to me actually...

Reply:
https://github.com/humanode-network/humanode/pull/1315#issuecomment-2474717145